micro_manager/micro_manager.py

```diff
@@ -67,8 +71,16 @@ def __init__(self, config_file: str) -> None:
         self._is_micro_solve_time_required = self._config.write_micro_solve_time()
 
         # Parameter for interpolation in case of a simulation crash
-        self._crash_threshold = 0.2
-        self._number_of_nearest_neighbors = 4
+        self._interpolate_crashed_sims = self._config.interpolate_crashed_micro_sim()
+        if self._interpolate_crashed_sims:
+            if Interpolation is None:
+                self._logger.info(
+                    "Interpolation is turned off as the required package is not installed."
+                )
+                self._interpolate_crashed_sims = False
+            else:
+                self._crash_threshold = 0.2
+                self._number_of_nearest_neighbors = 4
```

Comment:
Please write a comment here stating that these parameters could in the future be configurable.